includes/Classifai/Services/ImageProcessing.php

```diff
@@ -311,9 +311,16 @@ public function generate_image( WP_REST_Request $request ) {
 			return $provider;
 		}
 
+		$prompt = $request->get_param( 'prompt' );
+
+		// If our prompt exceeds the max length, throw an error.
+		if ( mb_strlen( $prompt ) > $provider->max_prompt_chars ) {
```

Comment:
@dkotter Most image processor providers will handle prompt length validation in the generate_image_callback function. We can remove it.